### ISSUE TITLE
on item 11

### ISSUE THREAD

**Issue by wanderer** [rosarior/Code-of-Merit#24]
> 11) If something is illegal outside the scope of the project, it is illegal in the scope of the project. This Code of Merit does not take precedence over governing law.

I don't understand why this item exists. I think it's outside the scope of this document. Opions?

**Comment by CartesianDuelist** [comment]
Agreed. You don't need to tell people to obey the law in a Code of Merit, i.e. the anti-code-of-conduct.

**Comment by Minoru** [comment]
> What I wanted to say is: "If something is illegal in real life it is not allowed in the project." as a blanket clause to avoid mentioning: threats, harassment, blackmailing or any other criminal act already codified by existing laws.

[Source](https://github.com/rosarior/Code-of-Merit/issues/12#issuecomment-172077339)

**Comment by fche** [comment]
That's not only a moot point, but introduces jurisdictional ambiguities & invites amateur law enforcement.  IMHO ditch the whole clause.

**Comment by wanderer** [comment]
>>What I wanted to say is: "If something is illegal in real life it is not allowed in the project." as a blanket clause to avoid mentioning: threats, harassment, blackmailing or any other criminal act already codified by existing laws.

if we apply the code of merit to itself. I think this is in conflict with item 10.

>There is no room for ambiguity: Ambiguity will be met with questioning; further ambiguity will be met with silence. It is the responsibility of the originator to provide requested context.

Futhermore if someone did try to blackmail someone else I think it would be in violation of item 9
> Discuss or debate the idea, not the person.

**Comment by CartesianDuelist** [comment]
Agreed entirely. Ditch it. It's pointless.

**Comment by rosarior** [comment]
If the reason to remove something is because it is obvious or "we don't need to tell people to obey the law" then not only this Code of Merit but basically every other regulatory document would need to be removed too.

The Code of Merit is not an anti-code-of-conduct. It defines how a project will be managed in a way to ensure technical capital increases as function of time. If someone wants to behave like a jackass, they are in their right. Project managers or core teams are also in their right to respond to that behavior as they see fit.

Blanket clauses are a safer bet than listing individual items. They includes everything. It is not the same as being ambiguous.  

I need a better reason to make removals than just: I don't like it, remove it.

**Comment by CartesianDuelist** [comment]
"If the reason to remove something is because it is obvious or "we don't need to tell people to obey the law" then not only this Code of Merit but basically every other regulatory document would need to be removed too."

Unlike the Code of Merit, the law is a social contract all are already expected to obey. It is a per-requisite of existing within society. It is also already covered by the terms of use on pretty much all platforms and server providers. 

That said, fche's post is the most poignant:

"That's not only a moot point, but introduces jurisdictional ambiguities & invites amateur law enforcement. IMHO ditch the whole clause."

**Comment by anatolyborodin** [comment]
So, if something is illigal in e.g. Saudi Arabia or North Korea, it's also automatically illegal in the scope of the project?

**Comment by Chiiruno** [comment]
Obeying the law is something out of scope of a project, and should be taken for granted, since it's not the project that will enforce it, but instead the governing body.
If you must keep it, it should be applied to the server or computer that the project is on, and not the project itself, since different countries and municipalities have different laws.
Just because North Korea says I can't say the word "I", or something, doesn't mean I, who am based in the USA, and the project, which may or may not be based in the USA, should have to take that into account.
This clause needs to be either altered or removed, IMO.

**Comment by gezeta-id** [comment]
"Application of this code does not exempt yourself or anyone from complying with any Applicable Laws."

And if you feel it needs some explanation about "Applicable Laws", the [Wikimedia Foundation](https://en.wikipedia.org/wiki/Wikipedia:Applicable_law) has a nice, clear and short explanation which could serve as inspiration. It's basically "any law that already applies to you and/or the project".
